src/Data/Camera/Filtering/ICameraFilter.cs

```diff
@@ -0,0 +1,11 @@
+namespace Data.Camera.Filtering
+{
+    using System.Linq;
+    using Business;
+    using Business.Camera.Fetching.Models;
+
+    public interface ICameraFilter : ICommand<IQueryable<ECamera>, IQueryable<ECamera>>
```

Comment:
Should we register this Interface on IoC ?

Reply:
Done